Apps/GatewayApi/src/Validations/PhoneNumberValidator.cs

```diff
@@ -0,0 +1,62 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.GatewayApi.Validations
+{
+    using System;
+    using FluentValidation;
+    using PhoneNumbers;
+
+    /// <summary>
+    /// Class encapsulating validation phone numbers leveraging Google's libphonenumber library.
+    /// </summary>
+    public class PhoneNumberValidator : AbstractValidator<string>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PhoneNumberValidator"/> class.
+        /// </summary>
+        /// <param name="region">The region that the phone number should be validated against.</param>
+        public PhoneNumberValidator(string region = "CA")
```

Comment:
I believe PHSA are using US to validate the numbers, do we allow only CA numbers or US include both?

Reply:
https://stackoverflow.com/questions/19682536/is-it-possible-to-programatically-distinguish-between-american-and-canadian-phon

I see that US is likely a safer bet to ensure that its a valid US/CA number, geocoding is done by a different process, so I think you're right to rather lean towards US.

I went with US and then second guessed my decision later on hence the dissonance in the documentation, I'm updating accordingly